src/EA.Iws.Web/Areas/AdminExportNotificationMovements/Views/CaptureMovement/Create.cshtml

```diff
@@ -37,17 +39,61 @@
             <button class="button" type="submit">@Constants.ContinueButtonText</button>
 
         }
+        @Html.ActionLink(Resource.ReturnToSummary, "Index", "Home")
+        </div>
+        <br/>
```

Comment:
Is a break tag necessary? I would have thought the columns would work fine without it.